libs/ledger-live-common/src/families/cosmos/specs.ts

```diff
@@ -437,10 +438,18 @@ const generateGenericCosmosTest = (currencyId: string, config?: Partial<AppSpec<
     appQuery: {
       model: DeviceModelId.nanoS,
       appName: "Cosmos",
+      appVersion: "2.34.11",
     },
     genericDeviceAction: acceptTransaction,
     testTimeout: 2 * 60 * 1000,
     test: cosmosLikeTest,
+    onSpeculosDeviceCreated:
+      currencyId === "injective"
+        ? async ({ transport }) => {
```

Comment:
may be we should add an inline comment here to explain the expert mode for injective here

Reply:
please check updated version